CONTRIBUTING.md

```diff
@@ -22,33 +22,49 @@ Here are some links to the documentation that could be helpful when contributing
 - GitLab (through `Python-Gitlab`)
   - [Python-Gitlab documentation](https://python-gitlab.readthedocs.io/)
   - for details also see [official GitLab API docs](https://docs.gitlab.com/ee/api/)
-- Pagure (through `requests`) - API is dependant on deployed version of Pagure service;
+- Pagure (through `requests`) - API is dependent on deployed version of Pagure service;
   `ogr` is majorly used on (links lead directly to API docs)
   - [src.fedoraproject.org](https://src.fedoraproject.org/api/0/)
   - [pagure.io](https://pagure.io/api/0/)
   - [git.stg.centos.org](https://git.stg.centos.org/api/0/)
 
 ## Testing
 
-Tests are stored in [tests](/tests) directory.
-
-We use [Tox](https://pypi.org/project/tox) with configuration in [tox.ini](tox.ini).
+Tests are stored in the [tests](/tests) directory.
 
 Running tests locally:
 
 ```
 make prepare-check && make check
 ```
 
-Or you can run them in a container:
+The following packages are installed by 'make prepare-check'
+if they are not installed already:
+
+    python3-flexmock
+    python3-requre
+
+Or you can run tests in a container:
 
+    make prepare-build
     make build
     make check-in-container
 
-The above builds a local image with all the dependencies using
-[ansible-bender](https://github.com/ansible-community/ansible-bender), and
-runs the tests in a container created from that image. `TEST_TARGET` can be
-set to select a subset of the tests.
+The following packages are installed by 'make prepare-build'
+if they are not installed already:
+
+    podman
+    ansible-bender
+
+The 'make build' command builds a local image with all the dependencies using
+[ansible-bender](https://github.com/ansible-community/ansible-bender);
+the 'make check-in-container' command runs the tests in a container created
+from that image. `TEST_TARGET` can be set to select a subset of the tests.
+
+NOTE that 'make build' uses approximately 7.1 GB of disk space; see /etc/containers/storage.conf
+for the location of this storage. It is typically /var/lib/containers/storage for root,
+$HOME/.local/share/containers/storage for non-root users. ('make check-in-container' does not use
+appreciably more storage.)
```

Comment:
I think this information could get out of date quickly, might vary from system to system, and most of it is a repetition of the podman/buildah documentation. But there is some value mentioning that building the container image will require a handful of GB of storage. Maybe mention just that?

Reply:
I acknowledge that the required disk storage, and its location, might
change over time.  But I think we have to start somewhere, and I think
we need both pieces of information; as a user, my first question after
learning that OGR 'make check' requires 7.1 GB of storage space would
be "WHERE do I need 7.1 GB free?"

Please note also that the buildah manpage itself recapitulates
information from the podman manpage:

buildah:
       --root value

       Storage root dir (default:  "/var/lib/containers/storage"  for  UID  0,
       "$HOME/.local/share/containers/storage"  for  other users) Default root
       dir is configured in /etc/containers/storage.conf

podman:
   --root=value

       Storage root dir in which data, including images, is  stored  (default:
       "/var/lib/containers/storage"  for  UID 0, "$HOME/.local/share/contain‐
       ers/storage"  for  other  users).   Default  root  dir  configured   in
       /etc/containers/storage.conf.

Reply:
Buildah and podman are sister projects, they track each other closely and share many of the developers. We don't do either, so duplicating information from their man pages instead of referencing it, creates the risk of ultimately giving incorrect or misleading guidance for future contributors of this project. In my view it is outside of the scope of this contribution guide to teach its readers about container tooling and how to configure it, especially if that tooling has appropriate documentation.

Please update the note to the following:

```
NOTE that 'make build' requires a handful of GB of storage. Consult `man containers-storage.conf` to learn more about container storage paths (usually, for rootless users)  and configuration.
```